tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_rocm5.3.2

```diff
@@ -0,0 +1,295 @@
+ARG BASEIMAGE=centos:7
+ARG POLICY=manylinux2014
+ARG PLATFORM=x86_64
+ARG DEVTOOLSET_ROOTPATH=
+ARG LD_LIBRARY_PATH_ARG=
+ARG PREPEND_PATH=
+
+# Same as rocm/dev-centos-7:5.3.2
+# We build ROCm5.3.2 dev centos docker image manually(stage base_image, line11~line102).
+# It's same as https://github.com/RadeonOpenCompute/ROCm-docker/blob/master/dev/Dockerfile-centos-7
+FROM $BASEIMAGE AS base_image
+ARG ROCM_VERSION=5.3.2
+ARG AMDGPU_VERSION=5.3.2
+# Base
+RUN yum -y install git java-1.8.0-openjdk python; yum clean all
```

Comment:
Here ";" means it will ignore all installation errors. Is it really what you want?

And who need Java 8?